internal/provisioner/objects/deployment/deployment.go

```diff
@@ -98,6 +99,11 @@ func DesiredDeployment(contour *model.Contour, image string) *appsv1.Deployment
 		fmt.Sprintf("--leader-election-resource-name=%s", contour.LeaderElectionLeaseName()),
 		fmt.Sprintf("--envoy-service-name=%s", contour.EnvoyServiceName()),
 	}
+
+	if contour.Spec.LogLevel == v1alpha1.DebugLog {
+		args = append(args, "--debug")
+	}
```

Comment:
Please add some unit test cases to cover this -- looks like you can do something similar to https://github.com/projectcontour/contour/blob/main/internal/provisioner/objects/deployment/deployment_test.go#L156.

Reply:
done